website/content/en/pre-docs/deploy-guide/_index.md

```diff
@@ -0,0 +1,165 @@
+---
+title: "Deployment Guide"
+linkTitle: "Deployment Guide"
+weight: 50
+---
+
+Introducing new software the modifies how Kubernetes operates is complex, and no single solution exists. 
+
+This guide describes how Karpenter interacts with external resources, and modifies cluster behavior. 
+
+## Resources Dependencies
+
+### Cluster on AWS
+
+Currently, Karpenter creates nodes (EC2 Instances) through the AWS cloud provider module, the only such module implemented. Clusters on other cloud providers are contemplated, but are not currently supported. 
+
+Theoretically, a control plane could be hosted separately from Karpenter provisioned AWS instances. However, this complicates identity and authorization substantially. The Karpenter controller (i.e., agent) would need authorization with AWS APIs to provision new instances, and permissions with the control plane to add the new nodes. 
+
+Additionally, the intricacies of securing control plane communication between clouds presents another significant problem. 

```

Comment:
This is a bit wordy and since we already tell people basically to "don't do this" couldn't we have this be a sentence that says.

>It is recommended to run Karpenter in the cluster where it will be managing nodes. This simplifies authentication, communication, and troubleshooting for cluster administrators.